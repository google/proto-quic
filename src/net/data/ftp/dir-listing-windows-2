05-18-09  11:07AM       <DIR>          beta
01-06-09  04:25PM       <DIR>          cdrom
01-06-09  02:38PM                  129 checkdownload.html
08-19-09  11:23AM       <DIR>          Digital_Media_Player
12-29-08  10:27PM       <DIR>          LiveUpdate
08-20-09  04:34PM       <DIR>          mb
01-06-09  02:38PM                83933 Mb.eng
11-04-09  03:42PM       <DIR>          misc
06-12-09  04:20PM                  462 Path.idx
12-30-08  07:41AM       <DIR>          PDA
01-06-09  02:38PM                 2625 Platform.idx
12-30-08  07:41AM       <DIR>          print
01-06-09  02:42PM                  458 Readme.txt
10-28-09  02:27PM       <DIR>          server
12-30-08  08:59AM       <DIR>          vga
02-03-09  04:42PM              1951823 VH203_FR.pdf
