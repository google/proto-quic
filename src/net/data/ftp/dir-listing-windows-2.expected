d
beta
-1
2009
5
18
11
7

d
cdrom
-1
2009
1
6
16
25

-
checkdownload.html
129
2009
1
6
14
38

d
Digital_Media_Player
-1
2009
8
19
11
23

d
LiveUpdate
-1
2008
12
29
22
27

d
mb
-1
2009
8
20
16
34

-
Mb.eng
83933
2009
1
6
14
38

d
misc
-1
2009
11
4
15
42

-
Path.idx
462
2009
6
12
16
20

d
PDA
-1
2008
12
30
7
41

-
Platform.idx
2625
2009
1
6
14
38

d
print
-1
2008
12
30
7
41

-
Readme.txt
458
2009
1
6
14
42

d
server
-1
2009
10
28
14
27

d
vga
-1
2008
12
30
8
59

-
VH203_FR.pdf
1951823
2009
2
3
16
42
